extensions/schema-compare/src/models/schemaCompareOptionsModel.ts

```diff
@@ -72,590 +71,62 @@ export class SchemaCompareOptionsModel {
 		return optionName !== undefined ? this.deploymentOptions.booleanOptionsDictionary[optionName.optionName].description : '';
 	}
 
-	//#region Schema Compare Objects
-	public objectTypeLabels: string[] = [
-		loc.Aggregates,
-		loc.ApplicationRoles,
-		loc.Assemblies,
-		loc.AssemblyFiles,
-		loc.AsymmetricKeys,
-		loc.BrokerPriorities,
-		loc.Certificates,
-		loc.ColumnEncryptionKeys,
-		loc.ColumnMasterKeys,
-		loc.Contracts,
-		loc.DatabaseOptions,
-		loc.DatabaseRoles,
-		loc.DatabaseTriggers,
-		loc.Defaults,
-		loc.ExtendedProperties,
-		loc.ExternalDataSources,
-		loc.ExternalFileFormats,
-		loc.ExternalStreams,
-		loc.ExternalStreamingJobs,
-		loc.ExternalTables,
-		loc.Filegroups,
-		loc.Files,
-		loc.FileTables,
-		loc.FullTextCatalogs,
-		loc.FullTextStoplists,
-		loc.MessageTypes,
-		loc.PartitionFunctions,
-		loc.PartitionSchemes,
-		loc.Permissions,
-		loc.Queues,
-		loc.RemoteServiceBindings,
-		loc.RoleMembership,
-		loc.Rules,
-		loc.ScalarValuedFunctions,
-		loc.SearchPropertyLists,
-		loc.SecurityPolicies,
-		loc.Sequences,
-		loc.Services,
-		loc.Signatures,
-		loc.StoredProcedures,
-		loc.SymmetricKeys,
-		loc.Synonyms,
-		loc.Tables,
-		loc.TableValuedFunctions,
-		loc.UserDefinedDataTypes,
-		loc.UserDefinedTableTypes,
-		loc.ClrUserDefinedTypes,
-		loc.Users,
-		loc.Views,
-		loc.XmlSchemaCollections,
-		loc.Audits,
-		loc.Credentials,
-		loc.CryptographicProviders,
-		loc.DatabaseAuditSpecifications,
-		loc.DatabaseEncryptionKeys,
-		loc.DatabaseScopedCredentials,
-		loc.Endpoints,
-		loc.ErrorMessages,
-		loc.EventNotifications,
-		loc.EventSessions,
-		loc.LinkedServerLogins,
-		loc.LinkedServers,
-		loc.Logins,
-		loc.MasterKeys,
-		loc.Routes,
-		loc.ServerAuditSpecifications,
-		loc.ServerRoleMembership,
-		loc.ServerRoles,
-		loc.ServerTriggers
-	].sort();
-
-	public getObjectsData(): string[][] {
-		let data = [];
-		this.objectsLookup = {};
-		this.objectTypeLabels.forEach(l => {
-			let checked: boolean = this.getSchemaCompareIncludedObjectsUtil(l);
-			data.push([checked, l]);
-			this.objectsLookup[l] = checked;
+	/*
+	 * Sets include object types option's checkbox values and property name to the includeObjectTypesLookup map
+	 */
+	public setIncludeObjectTypesLookup(): void {
+		Object.entries(this.deploymentOptions.objectTypesDictionary).forEach(option => {
```

Comment:
Nit: remove blank link (but many of these changes are overlapping, right?)

Reply:
@Benjin didn't get this comment! blank link?